### DISCUSSION TITLE
Ideas for server <-> client offline-first sync solutions

### DISCUSSION THREAD

**Discussion by quolpr** [trong-orm/trong-orm#22] [Ideas]
Hi! I plan to build the sync lib on top of trong-orm in the future. But for now, if anybody wants to build it by yourself there is some helpful information that may help you — https://github.com/trong-orm/trong-orm/issues/15#issuecomment-1171068460

**Comment by pietschy** [comment]
Hi Sergey,

This might be of interest on the CRDT front.  It's an SQLite loadable extension that adds CRDT support for tables.  Early days yet but looks promising.  

https://github.com/vlcn-io/cr-sqlite

**Reply by quolpr** [reply]
Yeah, I keep watching the progress. Looks very promising!
